src/chapters/sql-part2/studio.rst

```diff
@@ -0,0 +1,59 @@
+Studio: Library
+===============
+
+Word has gotten out about your amazing SQL skills!
+The head librarian at the local library asked if you could help with their database.
+
+Database Setup
+--------------
+
+Let's start by creating a new schema in MySQL Workbench called ``library``.
+Once you have accepted all defaults and hit "Apply", run the provided scripts to create tables and add data.
```

Comment:
The convention I've been following is to use italics for menu/button choices.

Reply:
Done!